dbt/adapters/athena/column.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+
+from dbt.adapters.athena.relation import TableType
+from dbt.adapters.base.column import Column
+from dbt.exceptions import DbtRuntimeError
+
+
+@dataclass
+class AthenaColumn(Column):
+    table_type: TableType = TableType.TABLE
+
+    def is_iceberg(self) -> bool:
+        return self.table_type == TableType.ICEBERG
```

Comment:
Not sure to get why we need this in the definition of the column (same goes for `table_type`) ?

Reply:
because is using is_iceberg https://github.com/dbt-athena/dbt-athena/pull/196/files/2bfa18e0f717cd0270790424b7e816fe16e6bc50#diff-786e4e9820eed125a3ea010af0b100bcf95e87aec14a18adbb84ecbcafc4e0a1R33 to pick the right timestamp type

Reply:
yes, @nicor88 has provided the correct answer

Reply:
it's not smth unique. For instance, [dbt-hive](https://github.com/cloudera/dbt-hive/blob/main/dbt/adapters/hive/column.py) uses table_type attribute for column as well :-)